templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete a Car</h1>
+  
+  <p>Are you sure you want to delete a car ?</p>
+  
+  <form action="" method="post">
+    {% csrf_token %}
+    
+    <input type="submit" class="btn btn-danger" value="Yes, delete">
```

Comment:
Please, add `"Cancel"` button as well as mentioned in a checklist for all deletion templates. Update corresponding screenshots